.github/workflows/merge.yml

```
@@ -15,6 +15,11 @@ on:
 permissions:
   contents: read # to fetch code (actions/checkout)
 
+env:
+  REPO_SLUG: "docker/compose"
```

```suggestion
  REPO_SLUG: "docker/compose-bin"
```

https://github.com/docker/compose/pull/9787#issuecomment-1226032270

---

We imagined to re-use the existing `docker/compose` repository to store the new images as the users should now use the docker official image to run Compose in a container

---

I think it should be another repo. https://hub.docker.com/r/docker/compose is already a runnable image. It will be confusing for users imo.

Best would be to deprecate https://hub.docker.com/r/docker/compose and update hub repo description that would tell to switch to the official docker image with embedded compose or use the `compose-bin` one if user wants to use its own base image like:

```dockerfile
FROM debian
COPY --from=docker/compose-bin /docker-compose /usr/bin/compose
RUN compose version
```

(as compose can be standalone too)

---

We'll use a dedicated repo ` https://hub.docker.com/r/docker/compose-bin` for the new images 👍